features-sdk/step_definitions/controllers-steps.js

```diff
@@ -165,3 +166,41 @@ Then('I got an error with id {string}', function (id) {
 
   assert(this.props.error.id === id, `Expected error to have id "${id}", but got "${this.props.error.id}"`);
 });
+
+When('I call the deprecated publicApi method', async function () {
```

Comment:
This step should use the generic [API action step](https://github.com/kuzzleio/kuzzle/blob/master/features-sdk/step_definitions/controllers-steps.js#L7)
You can modify this step to also store the entire `response` object in the `this.props` object